scripts/emulate_app.sh

```diff
@@ -23,6 +23,7 @@ function check_launch() {
 
 function check_boot() {
     boot_status=$(adb -s emulator-5554 shell getprop sys.boot_completed)
+    echo $boot_status
```

Comment:
the output from these check functions are `/dev/null` (silenced) so can we delete this line?